survey_calculator/README.rst

```diff
@@ -0,0 +1,76 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Survey Calculator
+=================
+
+This module was written to add a calculator that will compute results from
```

Comment:
Hi @Ehtaga 
Thanks for your contribution to the OCA. 

This module adds a calculator that will compute results from